app/timetables/models.py

```diff
@@ -1,3 +1,12 @@
 from __future__ import unicode_literals
 
 from django.db import models
+
+from app.shared.models import CommonInfo
+
+
+class Dish(CommonInfo):
+    name = models.CharField(max_length=60)
```

Comment:
@andela-joyebanji: I don't think field option _unique_ is True by default; it is desirable for this model not to have duplicate entries of dishes.